parser/Builder.cc

```diff
@@ -8,6 +8,7 @@
 #include "ruby_parser/diagnostic.hh"
 
 #include <algorithm>
+#include <iostream>
```

Comment:
please remove this

Reply:
This line is not there anymore, looks like your review is a bit outdated.